llmware/library.py

```diff
@@ -746,14 +1181,42 @@ def add_office(self, input_folder=None):
 
     def get_all_library_cards(self, account_name='llmware'):
 
-        """ Get all library cards for all libraries on account """
+        """ Get all library cards for all libraries on account 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Get all library cards for all libraries on account.
```